apps/src/applab/appStorage.js

```diff
@@ -16,6 +16,8 @@ var AppStorage = module.exports;
 function getStatusDescription(status) {
   if (status === 429) {
     return 'Rate limit exceeded.';
+  } else if (status === 413) {
+    return 'Storage limit exceeded';
```

Comment:
It would help to clarify in this error message that the individual item size was too large, as opposed to having exceeded an overall storage size limit. Maybe "Storage item size limit exceeded" ?

I'm kind of regretting extracting this method because now it makes it messier to give error messages which are specific to the combination of the block used and the http status, for example "the max record size is N" vs "the max key-value size is N"